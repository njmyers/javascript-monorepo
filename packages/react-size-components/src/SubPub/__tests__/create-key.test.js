import createKey from '../create-key';

describe('it works', () => {
  test('it is a function ', () => {
    expect(typeof createKey).toBe('function');
  });

  test('it runs without crashing', () => {
    expect(createKey()).toMatchSnapshot();
  });
});
