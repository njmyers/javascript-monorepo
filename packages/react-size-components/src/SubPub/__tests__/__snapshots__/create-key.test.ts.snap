// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works it runs without crashing 1`] = `"2e7c85bb"`;
